marketpulse/main/templates/fxosprice_all.html

```diff
@@ -2,7 +2,41 @@
 
 {% block content %}
   <h2 class="title">Firefox OS devices prices</h2>
-  <h3 class="title">Your contributions</h3>
+  <h3 class="title">Contributions</h3>
+
+  <div class="row">
+    <form action="" method="get">
+      <div class="col-md-5">
+        <div class="form-group">
+          <select class="form-control" name="country_code" >
+            <option value="">All Countries</option>
+            {% for country in countries %}
+              <option value="{{ country.code }}" {% if country.code == country_code %}selected{% endif %}>
+                {{ country.name }}
+              </option>
+            {% endfor %}
+          </select>
+        </div>
+      </div>
+      <div class="col-md-5">
+        <div class="form-group">
+          <select class="form-control" name="device_pk">
+            <option value="">All Devices</option>
+            {% for device in devices %}
+              <option value="{{ device.pk }}" {% if device.pk == device_pk %}selected{% endif %}>
+                {{ device }}
+              </option>
+            {% endfor %}
+          </select>
+        </div>
+      </div>
+      <div class="col-md-2">
+        <div class="form-group">
+          <button type="submit" class="btn btn-default btn-lg">Filter</button>
```

Comment:
Personally I would prefer the button to move on the right side. It's totally up to you, whether you move it or not.